Library/Homebrew/cleanup.rb

```diff
@@ -0,0 +1,126 @@
+require "bottles"
+require "formula"
+require "thread"
+
+module Homebrew
```

Comment:
We could do that but there’s already a `Cleaner` module; we need to find a name not to confuse both modules. Or maybe merge them?